api/lib/application/assessments/assessment-controller.js

```diff
@@ -114,6 +117,27 @@ module.exports = {
 
     return competenceEvaluationSerializer.serialize(competenceEvaluations);
   },
+
+  async autoValidateNextChallenge(request, h) {
+    const assessmentId = request.params.id;
+    const locale = extractLocaleFromRequest(request);
+    const assessment = await usecases.getAssessment({ assessmentId, locale });
+    const userId = assessment.userId;
+    const fakeAnswer = {
+      assessmentId,
+      challengeId: assessment.lastChallengeId,
+      value: 'fake_answer',
```

Comment:
```suggestion
      value: 'FAKE_ANSWER_WITH_AUTO_VALIDATE_NEXT_CHALLENGE',
```
Ca serait plus visible si besoin.

Reply:
Ce nouveau nommage plus explicite et plus visible a été ajouté avec un nouveau commit. Merci.